src/components/LinkBar/LinkbarView.js

```diff
@@ -29,7 +32,6 @@ const styles = () => ({
     fontFamily: 'Raleway',
     fontSize: '10px',
   },
-
 });
 
 export default withStyles(styles)(LinkBar);
```

Comment:
You can use default props in here 
LinkBar.defaultProps = {
  title: 'NCI Cancer Research Data Commons',
  url: 'https://datacommons.cancer.gov/?cid=crdcnav_hp_gdc.cancer.gov',
};

Reply:
Done